spec/factories/seeds/school_factory.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_school, class: "School") do
+    transient { location { Faker::Address.city } }
+    transient { domain { [location.parameterize, Faker::Internet.domain_suffix].join } }
+
+    urn { Faker::Number.unique.decimal_part(digits: 7).to_s }
+    name { [location, %w[High Grammar Infant Nursary].sample, "School"].join(" ") }
```

Comment:
As this was for seeds rather than tests I was aiming for realistic-looking data, so the school would be `Northampton High`, in `Northampton`, and the email address would be `head@northampton-high.net`. I was also thinking about 'cascading' the domain down to the users (`lucy-maths-teacher@northampton-high.net`) but that's one for another day!

I figured that some consistency might make navigating in review apps a bit more cohesive.

Reply:
Could this stuff be set in the scenarios, so basic Fakery by default but known values for scenarios?